package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "jest --verbose --collectCoverage",
-    "start": "node server/index.js",
+    "start": "nodemon server/index.js",
```

Comment:
I seem to forget to change this back to 'node' upon release to production. So it might be a good idea to make a second alias for nodemon then you can "npm run nodemon"